lib/cryptosuites/suites.go

```diff
@@ -189,6 +193,8 @@ var (
 		EC2InstanceConnect: Ed25519,
 		// GitHubProxyCASSH uses same algorithms as UserCASSH.
 		GitHubProxyCASSH: RSA2048,
+		// GitClient uses same algorithms as UserCASSH.
+		GitClient: RSA2048,
```

Comment:
Is there a reason why we're not defaulting to P256 or Ed25519 here? The comment for the `legacy` suite states "new features should always use the new algorithms".